tests/pytests/unit/modules/test_npm.py

```diff
@@ -0,0 +1,199 @@
+"""
+    :codeauthor: Jayesh Kariya <jayeshk@saltstack.com>
+
+    Test cases for salt.modules.npm
+"""
+
+
+import textwrap
+
+import pytest
+
+import salt.modules.npm as npm
+import salt.utils.json
+from salt.exceptions import CommandExecutionError
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    with patch("salt.modules.npm._check_valid_version", MagicMock(return_value=True)):
```

Comment:
@s0undt3ch is it okay to patch during this function?

Reply:
yes, it is.